tools/cli/Makefile

```
@@ -58,6 +60,11 @@ fix-lint: ## Fix linting errors
 list: ## List all make targets
 	@${MAKE} -pRrn : -f $(MAKEFILE_LIST) 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | egrep -v -e '^[^[:alnum:]]' -e '^$@$$' | sort
 
+.PHONY: unit-test
+unit-test: ## Run unit-tests
+	@echo "==> Running unit tests..."
+	$(TEST_CMD) -race -cover -count=1 -coverprofile $(COVERAGE) ./...
```

[non-blocking] if you want to start caching the tests
```suggestion
unit-test: ## Run unit-tests
	@echo "==> Running unit tests..."
	$(TEST_CMD) -race  ./...
```
but that will drop coverage but we don't really use that coverage